Too new cmake (e.g. 4.0.3) is being rejected

Hello,

```bash
> cmake --version
cmake version 4.0.3-dirty
> CMAKE_POLICY_VERSION_MINIMUM=3.5 node-gyp rebuild # This works
```

Without the env var it fails. This is a snippet from the build log.
```
150 error CMake Error at CMakeLists.txt:20 (CMAKE_MINIMUM_REQUIRED):
150 error   Compatibility with CMake < 3.5 has been removed from CMake.
150 error
150 error   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
150 error   to tell CMake that the project requires at least <min> but has been updated
150 error   to work with policies introduced by <max> or earlier.
150 error
150 error   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
150 error
150 error
150 error make[1]: *** No rule to make target 'sword_static'. Seis.
```

[edit]
I was a bit hasty.
```bash
> CMAKE_POLICY_VERSION_MINIMUM=3.5 npm i node-sword-interface # This works
```

--- a/scripts/build_sword.sh
+++ b/scripts/build_sword.sh
@@ -47,6 +47,10 @@ mkdir -p sword_build
 
 SWORD_BUILD_TYPE="Release"
 
+# Newer CMake versions have removed compatibility with CMake < 3.5.
+# SWORD is configured with CMake minimum 2.6.0, which leads to a build error with newer CMake versions (e.g. 4.0.3).
+export CMAKE_POLICY_VERSION_MINIMUM=3.5
+
 if [ "$1" = "--android" ] ; then
   git clone https://github.com/karlkleinpaste/biblesync.git
   git -C biblesync checkout 2.1.0

